plugins/networktables/src/main/java/edu/wpi/first/shuffleboard/plugin/networktables/util/NetworkTableUtils.java

```diff
@@ -52,7 +52,10 @@ public static DataType dataTypeForEntry(String key) {
       return DataTypes.Map;
     }
     if (rootTable.containsKey(normalKey)) {
-      return DataTypes.getDefault().forJavaType(rootTable.getEntry(normalKey).getValue().getValue().getClass()).get();
+      var ntValue = rootTable.getEntry(normalKey).getValue();
+      if (ntValue.isValid()) {
+        return DataTypes.getDefault().forJavaType(ntValue.getValue().getClass()).get();
+      }
```

Comment:
It feels like this shouldn’t fall through (to the if subtable comparison), and instead should immediately return null.